app/controllers/messages.rb

```diff
@@ -0,0 +1,25 @@
+class Chitter < Sinatra::Base
+
+  get '/messages' do
+    @messages = Message.all
+    erb :'messages/index'
+  end
+
+  get '/messages/new' do
+    unless session[:user_id].nil?
```

Comment:
Style/UnlessElse: Do not use `unless` with `else`. Rewrite these with the positive case first. (https://github.com/bbatsov/ruby-style-guide#no-else-with-unless)